src/Blubrry/REST/Resource/Social.php

```diff
@@ -49,6 +68,85 @@ public function updateListing($program_keyword, $body) {
     public function getSocial($program_keyword, $params) {
         $path = '/2/social/' . $program_keyword . '/get-social-options.json';
 
+        $social_options = array();
+        $social_options['social-options'] = array();
+
+        $social_types=array('twitter', 'youtube', 'facebook');
```

Comment:
@lucashillebrandt Spaces around the equal sign, and also use `['twitter'...]` instead of `array()`.